src/Core/AbstractTest.php

```diff
@@ -182,6 +182,30 @@ public function setPending($state)
     }
 
     /**
+     * Set the focused status of the test and its children according to the
+     * supplied focus pattern and/or skip pattern
+     *
+     * @param string|null $focusPattern
+     * @param string|null $skipPattern
+     */
+    public function applyFocusPatterns($focusPattern, $skipPattern = null)
+    {
```

Comment:
Yeah, that's the intent, at least. Ginkgo has a similar feature:

> When using the command line flags you can specify one or both of --focus and --skip. If both are specified the constraints will be ANDed together.

It's not really clear what they mean by "ANDed together", but I assume they meant "matched by --focus AND NOT matched by --skip". It seems like the most useful and intuitive option.